web/satellite/static/errors/500.css

```diff
@@ -0,0 +1,49 @@
+/* Copyright (C) 2019 Storj Labs, Inc. */
+/* See LICENSE for copying information. */
+
+.error-container {
```

Comment:
you should double check if this css file corresponds our new styles style guide
unfortunately, this file won't be checked by our linter